src/modm/ui/menu/choice_menu_impl.hpp

```diff
@@ -11,43 +12,48 @@
  */
 // ----------------------------------------------------------------------------
 
-#include "choice_menu.hpp"
+#ifndef CHOICE_MENU_HPP
+#	error 	"Don't include this file directly, use choice_menu.hpp instead!"
+#endif
 
+#include "abstract_view.hpp"
 
-modm::ChoiceMenu::ChoiceMenu(modm::ViewStack* stack, uint8_t identifier) :
-	modm::AbstractMenu(stack, identifier),
+template<typename Allocator>
+modm::ChoiceMenu<Allocator>::ChoiceMenu(modm::ViewStack<Allocator>* stack, uint8_t identifier) :
+	modm::AbstractMenu<Allocator>(stack, identifier),
 	display_update_time(500),
 	timer(display_update_time),
 	buttonAction(false),
 	title(""),
 	homePosition(0),
 	position(0)
 {
-	this->maximalDrawnEntrys = (getViewStack()->getDisplay().getHeight()- 16) / 8 ;
+	this->maximalDrawnEntrys = (this->getViewStack()->getDisplay().getHeight()- 16) / 8 ;
```

Comment:
Is `maximalDrawnEntrys` the max number of entries that fit on the screen? Might be good to clarify that